src/20250320_Multi_RiskStewardParameterUpdatesPhase3/RiskStewardParameterUpdatesPhase3_20250320.s.sol

```diff
@@ -0,0 +1,447 @@
+    actionsAvalanche[0] = GovV3Helpers.buildAction(
+      type(AaveV3Avalanche_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[2] = GovV3Helpers.buildAvalanchePayload(vm, actionsAvalanche);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsOptimism = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsOptimism[0] = GovV3Helpers.buildAction(
+      type(AaveV3Optimism_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[3] = GovV3Helpers.buildOptimismPayload(vm, actionsOptimism);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsArbitrum = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsArbitrum[0] = GovV3Helpers.buildAction(
+      type(AaveV3Arbitrum_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[4] = GovV3Helpers.buildArbitrumPayload(vm, actionsArbitrum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsMetis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsMetis[0] = GovV3Helpers.buildAction(
+      type(AaveV3Metis_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[5] = GovV3Helpers.buildMetisPayload(vm, actionsMetis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsGnosis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsGnosis[0] = GovV3Helpers.buildAction(
+      type(AaveV3Gnosis_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[6] = GovV3Helpers.buildGnosisPayload(vm, actionsGnosis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBase = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBase[0] = GovV3Helpers.buildAction(
+      type(AaveV3Base_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[7] = GovV3Helpers.buildBasePayload(vm, actionsBase);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsScroll = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsScroll[0] = GovV3Helpers.buildAction(
+      type(AaveV3Scroll_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[8] = GovV3Helpers.buildScrollPayload(vm, actionsScroll);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBNB = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBNB[0] = GovV3Helpers.buildAction(
+      type(AaveV3BNB_RiskStewardParameterUpdatesPhase3_20250320).creationCode
+    );
+    payloads[9] = GovV3Helpers.buildBNBPayload(vm, actionsBNB);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsZkSync = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsZkSync[0] = GovV3Helpers.buildActionZkSync(
+      vm,
+      'AaveV3ZkSync_RiskStewardParameterUpdatesPhase3_20250320'
+    );
+    payloads[10] = GovV3Helpers.buildZkSyncPayload(vm, actionsZkSync);
+
+    IPayloadsControllerCore.ExecutionAction[]
```

Comment:
For Linea, payload needs to be deployed and registered first, then add payloadId etc here.

example: https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20250228_Multi_ClinicStewardActivation/ClinicStewardActivation_20250228.s.sol#L387-L394

Reply:
I'm not sure to fully understand.
I pushed a fix, but I don't really know how to find the correct payloadIds to specify for Linea and ZKSync

Reply:
Currently foundry does not support to change the evm version while running a script.
This is problematic as create2 prediction on shanghai(which we use on ethereum scripts) will generate different bytecode then on linea(london), mantle & zksync.
Foundry atm, does not support running single script with multiple evms (https://github.com/foundry-rs/foundry/issues/9840#issue-2838059447)

Therefore with these networks currently the workaround is to:
1. wait for review
2. once it's reviewed, deploy and register the payload
3. take the id of the registered payload and hardcode it 🤷

Reply:
Okay, all clear thanks for the explanation 👌